src/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 add_library(GLPK)
+add_executable(GLPSOL)
 target_sources(GLPK PRIVATE glpk.h)
+target_sources(GLPSOL
+    PRIVATE
+        ../examples/glpsol.c
+    PRIVATE
+        glpk.h
+)
```

Comment:
no, should be placed in examples/CMakeLists.txt + depends on GLPK::GLPK
EDIT: done on master

Reply:
reverted